Sources/Common/Core/Math/index.js

```diff
@@ -1,6 +1,9 @@
 import seedrandom from 'seedrandom';
 import macro from 'vtk.js/Sources/macro';
 
+// eslint-disable-next-line import/no-cycle
```

Comment:
@finetjul , Even with this eslint-disable, the build failed. Do you know how to fix it ?

Reply:
don't you also have to set it in MatrixBuilder/index.js ?

Reply:
seems good. Thanks.